mujoco_py/mjrendercontext.pyx

```diff
@@ -26,6 +26,7 @@ cdef class MjRenderContext(object):
     cdef readonly object opengl_context
     cdef readonly int _visible
     cdef readonly list _markers
+    cdef readonly list _pixels
```

Comment:
this is a bit of a misnomer, and so is confusing. better name would be `_pixel_overlays`, `_draw_pixel_buffers` or something like that.